unix/vncpasswd/vncpasswd.man

```diff
@@ -9,11 +9,11 @@ vncpasswd \- change the VNC password
 .B vncpasswd
 allows you to set the password used to access VNC desktops.  Its default
 behavior is to prompt for a VNC password and then store an obfuscated version
-of this password to \fIpasswd-file\fR (or to $HOME/.vnc/passwd if no password
-file is specified.)  The \fBvncserver\fP script runs \fBvncpasswd\fP the first
-time you start a VNC desktop, and it invokes \fBXvnc\fP with the appropriate
-\fB\-rfbauth\fP option.  \fBvncviewer\fP can also be given a password file to
-use via the \fB\-passwd\fP option.
+of this password to \fIpasswd-file\fR (or to \fB$XDG_CONFIG_HOME/tigervnc/passwd\fP
```

Comment:
The man page style is italic for file paths.

(For reference, the common style is documented as its own man page: https://man7.org/linux/man-pages/man7/man-pages.7.html)